src/main/kotlin/me/sargunvohra/lib/pokekotlin/model/resource.kt

```diff
@@ -1,20 +1,40 @@
 package me.sargunvohra.lib.pokekotlin.model
 
+private fun urlToId(url: String): Int {
+    return "/-?[0-9]+/$".toRegex().find(url)!!.value.filter { it.isDigit() || it == '-' }.toInt()
+}
+
+private fun urlToCat(url: String): String {
```

Comment:
Would prefer to use the full word `urlToCategory` here

Reply:
Happy to change it if you'd like, but I literally just copied it from `adapters.kt`

Reply:
Lol oops, don't worry about it